draft-scenarios/clash/arcane_artillery.tex

```diff
@@ -0,0 +1,155 @@
+  \item 5 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile that contains an Obelisk. If not enough are available, treat Witch Huts as Obelisks
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\textbf{For a 6-player Scenario:}
+\begin{itemize}
+  \item 6 × Starting (I) Map Tile
+  \item 6 × Near (IV--V) Map Tile
+  \item 1 × Center (VI--VII) Map Tile, which must contain the Dragon Utopia Field
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+To win the Scenario, a Hero must Flag the Dragon Utopia Field.
+
+\vspace*{\fill}
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+At the end of the \nth{12} Round, if there is no winner, all players lose the Scenario.
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{6} Round:}
+\begin{itemize}
+  \item Remove all Black cubes from every Tile.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+Before the start of this Scenario:
+
+\begin{itemize}
+  \item If playing Alliance mode, allies and oppoents choose starting Tiles alternately.
+\end{itemize}
+
+During this Scenario:
+
+\begin{itemize}
+  \item Heroes cannot enter starting Tiles other than their own.
+  \item A player may not use the Diplomacy card to skip Combat on the Dragon Utopia Field.
+  \item Arcane Spikes are represented by cubes not used by any player's Faction.
+\end{itemize}
+
+\vspace*{\fill}
+
+\columnbreak
+
+\begin{itemize}
+    \item Obelisks may be captured. The first time an Obelisk is captured, gain one Arcane Spike.
+    \item At the start of each player's turn, that player removes any unspent Arcane Spikes, then gains Arcane Spikes equal to the number of captured Obelisks they have.
+\end{itemize}
+
+\vspace{1em}
+
+\includegraphics[width=\linewidth, keepaspectratio]{\art/griffin.png}
+
+\end{multicols}
+
+\vspace{1em}
+
+\hommtable[]{15}{
+  \begin{tabularx}{\linewidth}{X}
+  \darkcell{\raisebox{-.2\height}{} \textbf{Activation Ability: Arcane Artillery}}\\
+  \lightcell[4.8]{\raisebox{-.25\height}{} \includegraphics[height=10px]{\layout/listdot.png} Deal 1 \svg{damage-table} to target Unit by placing an Arcane Spike on it. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery ignores the Azure Dragon's special ability. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery does not count toward combat Round spell limit. \\ \includegraphics[height=10px]{\layout/listdot.png} Arcane artillery is considered to be a spell of every school of magic. \\ \includegraphics[height=10px]{\layout/listdot.png} Players may cast Arcane Artillery when an opponent is in combat with neutral Units, on the neutral Units turn. \\ \includegraphics[height=10px]{\layout/listdot.png} Each player may only cast Arcane Artillery once per combat Round. \\ \includegraphics[height=10px]{\layout/listdot.png} If playing Alliance mode, allies may cast Arcane Artillery on allied player's Unit activation.} \\
```

Comment:
I suggest making individual cells for every rule, like here:
![image](https://github.com/user-attachments/assets/bf023020-fa87-425d-93f0-beff7c8f4f0e)

Reply:
Also, please split every row to a separate line.